cpp/skyweaver/src/ObservationHeader.cpp

```diff
@@ -1,29 +1,53 @@
 
 #include "skyweaver/ObservationHeader.hpp"
-#include <type_traits>
+
 #include <cmath>
+#include <type_traits>
 namespace skyweaver
 {
+
+std::vector<float> parse_float_list(std::string const& str)
+{
+    std::vector<float> values;
+    std::size_t start = 0;
+    std::size_t end = 0;
+    while(end != std::string::npos) {
+        end = str.find(',', start);
+        values.push_back(std::stof(str.substr(start, end - start)));
```

Comment:
This could use some error handling, as the stof doesn't throw (see comments on hennings PR).